.circleci/config.yml

```diff
@@ -110,12 +110,9 @@ commands:
             if [[ $CIRCLE_TAG =~ $pat ]]; then
               echo "this is rc version $CIRCLE_TAG"
               rc_num=$(echo $CIRCLE_TAG | cut -d '-' -f 2)
-              if [[ << parameters.os-network >> =~ .*"darwin".* ]]; then
```

Comment:
@blukat29 Can you remove the `os-network` parameters in `pre-build`? It doesn't needed anymore.

Also, `os-network` parameter in darwin-executor doesn't work. Instead, it should be explicitly configure that parameter in each job which uses that parameter:`build-packaging`. It is wrongly coded before, so it needs to be fixed. You can fix it here, or I'll put it up in another PR.

Reply:
Is 611bc996 what you need?

Reply:
Yes, the code looks fine.

Reply:
OK. let me test it out.